### PR TITLE
Wfinfo locale

### PR DESCRIPTION
**Pull request by zoo-hyeon**
Created a language selection combo in the settings window.
TesseractEngine is Dispose and regenerated based on language selection.
I changed from creating a market item from filterAllJSON to an API call. This may not be efficient, but i developed it this way to support other languages ​​dynamically.

Legacy theme calculations changed.
I checked the normal operation in English and Korean, but I commented that a check is necessary.
And when the master branch was pulled, an error occurred because the Spawn Full Screen Reminder method was not implemented. Although it was temporarily processed, it needs to be corrected because i do not know what kind of screen it is intended for.

### CODE REVIEW

**Comment by zoo-hyeon** [comment]
During development, some garbage was not cleaned up.
This is an unnecessary white space replacement.

https://github.com/zoo-hyeon/WFinfo/blob/7901b61bcad050acd77224fde78b50c44b02dcfa/WFInfo/Ocr.cs#L567
https://github.com/zoo-hyeon/WFinfo/blob/7901b61bcad050acd77224fde78b50c44b02dcfa/WFInfo/Ocr.cs#L1380

**Comment by dapal-003** [comment]
I've gone through it and applied some fixes/changes. I'm happy with the current state of the Korean implementation quality. However there are still two things that need to be changed in OCR.CS . 
817 has a minimum part length requirement in characters. What's the lowest length that a korean part could have?
921 has a minimum word length, seeing as korean characters are often recognized as just one or two characters I've defaulted this to 2.

File: WFInfo/Ocr.cs [hunk 1]
```diff
@@ -918,7 +918,7 @@ private static List<InventoryItem> FindAllParts(Bitmap filteredImage)
                                             continue;
                                         }
                                     }
-                                    else if (currentWord.Length < 2)
+                                    else if (currentWord.Length < 2 && Settings.locale == "en")
```
Comment (dapal-003) [inline WFInfo/Ocr.cs:921]: Add a minimum Korean word length. Note the OCR tends to only highlight one or two charters in Korean.

File: WFInfo/Ocr.cs [hunk 2]
```diff
@@ -812,10 +812,10 @@ internal static void ProcessSnapIt(Bitmap snapItImage, Bitmap fullShot, Point sn
             snapItImageFiltered.Dispose();
             if (!File.Exists(applicationDirectory + @"\export " + DateTime.UtcNow.ToString("yyyy-MM-dd", Main.culture) + ".csv") && Settings.SnapitExport)
                 csv += "ItemName,Plat,Ducats,Volume,Vaulted,Owned," + DateTime.UtcNow.ToString("yyyy-MM-dd", Main.culture) + Environment.NewLine;
-
             foreach (var part in foundParts)
             {
-                if (part.Name.Length < 13) // if part name is smaller than "Bo prime handle" skip current part
+                if (part.Name.Length < 13 && Settings.locale == "en" || part.Name.Length < 2 && Settings.locale == "ko") // if part name is smaller than "Bo prime handle" skip current part
```
Comment (dapal-003) [inline WFInfo/Ocr.cs:817]: Add a minimum Korean prime part word length. Currently I've gone for two as I do not know the minimum. As close to the actual minimum is preferred.

**Comment by dimon222** [comment]
If you get a spare moment, try to reference tessdata `kor` in this `libs` branch
https://github.com/WFCD/WFinfo/pull/151/files#diff-e059b0cfe2a011db54d0c7c80e90f9573f562f06ddf61a23c0dc6809debdb786R21

**Pull request merged**